api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -152,7 +156,107 @@ public void saveUser_shouldShouldCreateUserWhoIsPatientAlready() throws SQLExcep
 		List<Patient> allPatientsSet = Context.getPatientService().getAllPatients();
 		assertEquals(1, allPatientsSet.size());
 	}
-	
+
+	@Test(expected = APIException.class)
+	public void createUser_shouldNotAllowExistingUser() {
+		UserService userService = Context.getUserService();
+		User someUser = getBrunoUser(userService);
+
+		userService.createUser(someUser, SOME_VALID_PASSWORD);
+	}
+
+	@Test(expected = APIException.class)
+	public void createUser_shouldNotAllowBlankPassword() {
+		UserService userService = Context.getUserService();
```

Comment:
since `userService` is used in every test, can you make it a class member and initialize it in an `@Before setUp()`